### PR TITLE
change version requirement

### PR DESCRIPTION
**Pull request by Deflaimun**
Resolves https://github.com/redpanda-data/documentation-private/issues/2062

### CODE REVIEW

File: modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc [hunk 1]
```diff
@@ -6,9 +6,9 @@ This topic provides the requirements and recommendations for provisioning Kubern
 
 == Operating system
 
-- Minimum recommended version of RHEL/CentOS: {supported-rhel-version}
+- Minimum version required of RHEL of RHEL/CentOS: {supported-rhel-required}. *Recommended*: {supported-rhel-recommended}
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc]:
```suggestion
- Minimum version required of RHEL/CentOS: {supported-rhel-required}. *Recommended*: {supported-rhel-recommended}
```

File: modules/deploy/pages/deployment-option/self-hosted/manual/production/dev-deployment.adoc [hunk 2]
```diff
@@ -19,8 +19,8 @@ See xref:get-started:quick-start.adoc[Redpanda Quickstart] to try out Redpanda i
 The same prerequisites for a production environment are recommended for a development environment.
 
 * Operating system
-** Minimum recommended version of RHEL of RHEL/CentOS: {supported-rhel-version}
-** Minimum version of Ubuntu: {supported-ubuntu-version}
+** Minimum version required of RHEL of RHEL/CentOS: {supported-rhel-required}. *Recommended*: {supported-rhel-recommended}
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/self-hosted/manual/production/dev-deployment.adoc]:
```suggestion
** Minimum version required of RHEL/CentOS: {supported-rhel-required}. *Recommended*: {supported-rhel-recommended}
```

File: modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment.adoc [hunk 3]
```diff
@@ -17,8 +17,8 @@ You can deploy Redpanda for production with a default deployment, which uses rec
 Follow these requirements and recommendations:
 
 * Operating system
-** Minimum recommended version of RHEL of RHEL/CentOS: {supported-rhel-version}
-** Minimum version of Ubuntu: {supported-ubuntu-version}
+** Minimum version required of RHEL of RHEL/CentOS: {supported-rhel-required}. *Recommended*: {supported-rhel-recommended}
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment.adoc]:
```suggestion
** Minimum version required of RHEL/CentOS: {supported-rhel-required}. *Recommended*: {supported-rhel-recommended}
```

**Review by Feediver1 — APPROVED** [review]
Pretty sure you have duplicate text in there for RHEL--I could be wrong. ?

**Comment by Deflaimun** [comment]
> Pretty sure you have duplicate text in there for RHEL--I could be wrong. ?

Yes I think the replacement for all files must have picked something more. Thanks for checking

**Review by mattschumpert — APPROVED** [review]

**Comment by mattschumpert** [comment]
@Deflaimun we now also have the same content in 'deploy for development' (duplicated), so that also needs updating

**Comment by Deflaimun** [comment]
Hi @mattschumpert. It's covered in this update. Kubernetes too.

**Comment by Deflaimun** [comment]
Here's all previews
https://deploy-preview-89--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements
https://deploy-preview-89--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/manual/production/dev-deployment
https://deploy-preview-89--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/manual/production/production-deployment

**Pull request merged**